ci-automation/garbage_collect.sh

```diff
@@ -89,6 +89,7 @@ function garbage_collect() {
         else
             echo "## ${version} is an OS image version. ##"
             rmpat="${BUILDCACHE_PATH_PREFIX}/containers/${os_docker_vernum}/flatcar-packages-*"
+            rmpat="${BUILDCACHE_PATH_PREFIX}/binpkgs/boards/*/${os_docker_vernum}/*"
```

Comment:
Is `${rmpat}` missing here? This line clobbers the previous setting.

```suggestion
            rmpat="${rmpat} ${BUILDCACHE_PATH_PREFIX}/binpkgs/boards/*/${os_docker_vernum}/*"
```

Reply:
Yes, good catch! copy+paste error. However, we need to remove the newline.

Reply:
Oh wait, there is no newline, Github UI just thinks it's clever to add a line break. Meh.